### PR TITLE
Fill account storage from config file

### PR DESCRIPTION
**Pull request by makxenov**
In addition all JSON-related functionality is moved to separate library.

### CODE REVIEW

File: bin/assigner/src/main.cpp [hunk 1]
```diff
@@ -130,18 +132,42 @@ int curve_dependent_main(const std::string& input_block_file_name,
...
+        }
+        auto init_err = init_account_storage(account_storage, asc_stream);
+        if (init_err) {
+            std::cerr << init_err.value() << std::endl;
+            return -1;
+        }
+
+        BOOST_LOG_TRIVIAL(debug) << "Account storage initialized with " << account_storage.size()
+                                 << " accounts: \n";
+        for (const auto& [addr, acc] : account_storage) {
```
[the remark this answers is not in the document]
Comment (makxenov) [reply bin/assigner/src/main.cpp]: Thanks, added

File: bin/assigner/src/state_parser.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            size_t balance_value = json_helpers::get_number(elem.value());
+            // Convert to big endian number as it required in evmc
+            boost::endian::native_to_big_inplace(balance_value);
+            // Set last bytes with balance value and others with zeros
+            constexpr size_t zero_padding = sizeof(account.balance.bytes) - sizeof(balance_value);
+            std::memcpy(account.balance.bytes + zero_padding, &balance_value,
+                        sizeof(balance_value));
+            std::memset(account.balance.bytes, 0, zero_padding);
+        } else if (elem.key() == "account") {
+            // Already processed
```
Comment (akokoshn) [inline bin/assigner/src/state_parser.cpp:60]: according state_schema, account json_obj, should not contain `key = account`, or i miss something?

Comment (makxenov) [reply bin/assigner/src/state_parser.cpp:60]: Sure, sorry, it must be `address` here

Comment (makxenov) [reply bin/assigner/src/state_parser.cpp:60]: Fixed

**Review by akokoshn — APPROVED** [review]

**Pull request merged**